src/net.h

```diff
@@ -861,6 +861,12 @@ class CNode
         return m_conn_type == ConnectionType::INBOUND;
     }
 
+    /* Whether we send addr messages over this connection */
+    bool RelayAddrsWithConn() const
```

Comment:
nit: Would be a bit clearer IMO to use the `Is` prefix or similar to distinguish between the interrogative and the imperative.

Reply:
Not my work: see #19724 this is based on.